elements/link.html

```diff
@@ -40,7 +40,8 @@
     <code>
       <strong>&lt;html&gt;</strong>
         &lt;head&gt;
-          &lt;title&gt;Document title&lt;/title&gt;
+        &lt;link rel="stylesheet" href="//code.jquery.com/ui/1.11.4/themes/smoothness/jquery-ui.css"&gt;
+        &lt;link type='text/css' href='css/style.css' rel='stylesheet' /&gt;
```

Comment:
This commit adds one link with attribute values in single quotes and another with them double quotes. Strive for consistency in style, especially within a particular file.